static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -250,6 +250,15 @@ export function DraggableTabBar({
         querySort: tempTab.querySort,
       };
       const newTabs = [...tabs, newTab];
+      navigate({
```

Comment:
Probably want to replace here, don't want to go to the temp ID when you click the browser back button